src/app/shared/components/notification-container/notification-container.component.ts

```diff
@@ -0,0 +1,147 @@
+      .subscribe((notification: Notification) => this.createNotification(notification))
+
+    this.statusMessageEventSubscription = this.statusMessageEventObserver.subscribeToStatusMessageEvents(statusMessageEvent =>
+      this.createNotificationFromStatusMessage(statusMessageEvent.statusMessage)
+    )
+
+    this.systemInformationService.getStatusMessages().subscribe(statusMessages => this.createNotificationsFromStatusMessages(statusMessages))
+  }
+
+  private createNotification(notification: Notification, elementId?: string): void {
+    const notificationElement: HTMLElement = this.createNotificationElement(notification, elementId)
+    this.insertNotificationIntoContainer(notificationElement)
+    if (!notification.isPersistent) {
+      this.setRemoveNotificationTimer(notificationElement)
+    }
+    this.moveNotifications()
+  }
+
+  private createNotificationElement(notification: Notification, elementId?: string): HTMLElement {
+    const notificationComponentRef: ComponentRef<NotificationComponent> = createComponent(NotificationComponent, {
+      environmentInjector: this.applicationRef.injector,
+    })
+
+    notificationComponentRef.instance.notification = notification
+    notificationComponentRef.instance.onRemoveCallback = (notificationElement: HTMLElement): void => this.removeNotificationElement(notificationElement)
+    notificationComponentRef.changeDetectorRef.detectChanges()
+
+    const notificationElement: HTMLElement = (<EmbeddedViewRef<NotificationComponent>>notificationComponentRef.hostView).rootNodes[0]
+    notificationElement.id = elementId ?? NON_PERSISTENT_NOTIFICATION_ID
+    return notificationElement
+  }
+
+  private insertNotificationIntoContainer(notificationElement: HTMLElement): void {
+    const host: HTMLElement = this.elementRef.nativeElement
+    host.insertAdjacentElement('afterbegin', notificationElement)
+    this.notificationElements.unshift(notificationElement)
+  }
+
+  private setRemoveNotificationTimer(notificationElement: HTMLElement): void {
+    setTimeout(() => this.removeNotificationElement(notificationElement), NOTIFICATION_DURATION_MS)
+  }
+
+  private removeNotificationElement(notificationElement: HTMLElement): void {
+    notificationElement.remove()
+    const index: number = this.notificationElements.indexOf(notificationElement)
+    if (index === -1) {
+      return
+    }
+    this.notificationElements.splice(index, 1)
+
+    this.moveNotifications()
+  }
+
+  private moveNotifications(): void {
+    if (this.notificationElements.length === 0) {
+      return
+    }
+    this.notificationElements[0].style.top = 0 + PIXEL_POSTFIX
+    let top: number = this.notificationElements[0].offsetHeight + STACK_GAP
+    for (let i = 1; i < this.notificationElements.length; i++) {
+      const notificationElement: HTMLElement = this.notificationElements[i]
+      notificationElement.style.top = top + PIXEL_POSTFIX
+      top += notificationElement.offsetHeight + STACK_GAP
+    }
+  }
```

Comment:
It would be nice to just have it as a list. However, I fell short of getting the entries in the list to animate up and down because when placed in the list and render by *ngFor the elements just "jumps" from one position to the next.

If you have a suggestion for doing that, then I am all ears :D (Currently the entries in the panel don't animate because I have them in a list instead of manaully inserting them into the DOM)